packages/client/src/createTRPCClientProxy.ts

```diff
@@ -93,11 +93,17 @@ const clientCallTypeMap: Record<
 export function createTRPCClientProxy<TRouter extends AnyRouter>(
   client: Client<TRouter>,
 ) {
-  const proxy = createProxy(({ path, args }) => {
+  const proxy = createProxy((path, args) => {
```

Comment:
Spread out args are more performant, as we aren't allocating a new object every time.